### ISSUE TITLE
Add components inside markdown

### ISSUE THREAD

**Issue by johan-smits** [rambip/leptos-markdown#2]
Is it possible like with Sycamore to add components in markdown that are reactive?

https://lukechu10.github.io/mdsycx/

**Comment by rambip** [comment]
Wow, `mdsycx` looks really cool, thank's for the link !

I didn't consider that, but it could be a cool feature for sure

**Comment by rambip** [comment]
It may be possible by passing a HashMap of signals as a prop for example.

What do you think would be the best syntax ?
Something like that maybe ?
```md
# Reactive markdown:
<signal>name_of_my_signal</signal>
```

**Comment by johan-smits** [comment]
That is one solution part of the solution. By passing a component like `<Counter initial="42" />` the `Counter` component could have implemented actions to increase or decrease like any other component in the view tree. With this you could for example embed a complete app inside your markdown. The feature is more powerful then just having a reactive signal in them.

I can't find his example of the counter but it worked like I described above.

**Comment by rambip** [comment]
This would require an analysis of the markdown at compile time right ?
I don'y see how that's possible otherwise
If it needs a proc-macro, I will not implement this feature.

**Comment by johan-smits** [comment]
Having a look at the code it uses a macro. Since Sycamore al very identical to Leptos it should be simple to port. Here is the [macro](https://github.com/lukechu10/mdsycx/blob/master/mdsycx-macro/src/from_md.rs)

**Comment by rambip** [comment]
I just realized that would involve writting a proc-macro to implement `FromMd`

from the mdsycx docs:
```rust
#[derive(Prop, FromMd)]
pub struct MyProps<'a, G: Html> {
    /// A prop that can be set from markdown.
    special_number: i32,
    /// The `children` prop is always required.
    children: Children<'a, G>,
}
```

**Comment by rambip** [comment]
Closed by 366fe7e

Note that it is still an experimental feature
